### DISCUSSION TITLE
Network microburst prevention

### DISCUSSION THREAD

**Discussion by thehhmdb** [LizardByte/community#362] [Sunshine]
Would it be possible to build something into Sunshine that would help with packet loss due to microbursts?

This can easily happen when the host is on a faster link speed than the client and the buffers on the in-between device like a switch are filled.

e.g.
Host 10Gb NIC => 10Gb Switch => Client 1Gb NIC

Maybe some sort of traffic shaper/limiter in Sunshine could help? or maybe allowing a slower 2nd network adapter to be used on the host instead of the main one?

**Comment by dizzydre21** [comment]
> Would it be possible to build something into Sunshine that would help with packet loss due to microbursts?
> 
> This can easily happen when the host is on a faster link speed than the client and the buffers on the in-between device like a switch are filled.
> 
> e.g. Host 10Gb NIC => 10Gb Switch => Client 1Gb NIC
> 
> Maybe some sort of traffic shaper/limiter in Sunshine could help? or maybe allowing a slower 2nd network adapter to be used on the host instead of the main one?

Did you ever get anywhere with this? I'm having extreme frame drops with multiple brands of 10Gb NICs, when the clients are 2.5Gb or 1Gb. I'm thinking this might be the issue.
